CODING_GUIDELINES.md

```diff
@@ -232,36 +232,61 @@ class MyClass
 ```
 
 ### 7. Header inclusion order. ###
-The headers should be placed in the following order:
- 1. Module header (in .cpp)
- 2. System/Qt/Boost etc. headers (split in subcategories if you have many).
- 3. Application headers, starting from *Base* headers.
+The headers should be placed in the following group order:
+  1. Module header (in .cpp)
+  2. C++ Standard Library headers
+  3. Boost library headers
+  4. Libtorrent headers
+  5. System headers
```

Comment:
Shouldn't it be immediately after C/C++ headers?

Reply:
yes, fixed.